### PR TITLE
[model] feat: update seed_oss v5

### PR DESCRIPTION
**Pull request by Coach257**
1. support v5 seed oss. #662 
2. remove duplicate v4 ci that tested under v5 env.
3. remove v4 moe model ckpt converter.

### CODE REVIEW

File: veomni/models/transformers/seed_oss/seed_oss_gpu_patch_gen_config.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+                vocab_size=self.config.vocab_size,
+                hidden_states=hidden_states,
+                weights=self.lm_head.weight,
+                **kwargs,
+            )
+        else:
+            logits = self.lm_head(hidden_states)
+            loss, _, log_probs, entropy = self.loss_function(
+                logits=logits, labels=labels, vocab_size=self.config.vocab_size, **kwargs
+            )
```
[the remark this answers is not in the document]
Comment (Coach257) [reply veomni/models/transformers/seed_oss/seed_oss_gpu_patch_gen_config.py:167]: install_loss_mapping patches the loss_function

File: veomni/models/transformers/seed_oss/seed_oss_npu_patch_gen_config.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+                vocab_size=self.config.vocab_size,
+                hidden_states=hidden_states,
+                weights=self.lm_head.weight,
+                **kwargs,
+            )
+        else:
+            logits = self.lm_head(hidden_states)
+            loss, _, log_probs, entropy = self.loss_function(
+                logits=logits, labels=labels, vocab_size=self.config.vocab_size, **kwargs
+            )
```
[the remark this answers is not in the document]
Comment (Coach257) [reply veomni/models/transformers/seed_oss/seed_oss_npu_patch_gen_config.py:156]: install_loss_mapping patches the loss_function

**Comment by Luosuu** [comment]
maybe keep the v4 moe models until #751 is merged?

**Comment by Coach257** [comment]
> maybe keep the v4 moe models until #751 is merged?

Sure, I'll keep the v4 modeling, but just remove the ci tests for v4?

**Review by FoolPlayer — APPROVED** [review]
